src/Mod/Spreadsheet/Gui/Command.cpp

```diff
@@ -279,8 +279,8 @@ CmdSpreadsheetAlignLeft::CmdSpreadsheetAlignLeft()
 {
     sAppModule = "Spreadsheet";
     sGroup = QT_TR_NOOP("Spreadsheet");
-    sMenuText = QT_TR_NOOP("Align &left");
-    sToolTipText = QT_TR_NOOP("Left-align contents of selected cells");
+    sMenuText = QT_TR_NOOP("Align &Left");
+    sToolTipText = QT_TR_NOOP("Left-aligns the contents of the selected cells");

```

Comment:
```suggestion
    sToolTipText = QT_TR_NOOP("Aligns cell contents to the left");
```
I think this is easier to read